src/main/webapp/script.js

```diff
@@ -557,6 +571,75 @@ function highlightDiff(cy, mutList, reason = "") {
   };
 }
 
+/**
+ * Highlights the change in tokens of the given node based on the token mutation. If 
+ * tokens are added to the node, it adds them to the tippy in green. If tokens are deleted,
+ * it adds them to the tippy in red.
+ * @param node whose tokens have changed
+ * @param tokenMut the mutation made to the tokens of the node
+ */
+function indicateChangedTokens(node, tokenMut) {
+  const type = tokenMut["type_"] || -1;
+  const tokens = tokenMut["tokenName_"];
+
+  if (!type || !tokens || !node.tip) {
+    return;
+  }
+
+  // Get the div that contains the tooltip content
+  let tipContent = node.tip.popperChildren.content.firstChild;
+  switch (type) {
+    case 1:
+      // add tokens
+      // If this node already has an "Added Token" list, get it
+      let addedList = tipContent.querySelector(`#${node.data().id}-added`);
```

Comment:
I wonder if it makes more sense to also reset the addedList and deletedList, i.e. if this mutation didn't mutate any tokens, the tooltip should also have empty "added tokens" and "deleted tokens" sections. What do you think?

Reply:
Since nodes which were mutated are highlighted, do you think adding empty lists would be useful?

Reply:
What I meant was, if node A's token were mutated at mutation N, I would expect to see the  "Added tokens" and "Deleted tokens" in the tooltip. But on mutation N+1, if I click on node A again I shouldn't see any added or deleted tokens.

Reply:
Actually I think I'm mistaken here. I got confused by "If this node already has an "Added Token" list". The tippy is always newly created for each node, right? So shouldn't it always only contain a token list section before this function is called? Why would the tippy already contain an "Added Token" list?

Reply:
Actually I think I'm mistaken here. I got confused by "If this node already has an "Added Token" list". The tippy is always newly created for each node, right? So shouldn't it always only contain a token list section before this function is called? Why would the tippy already contain an "Added Token" list?

Reply:
I was just considering the case where there are two change token mutations for a node in the multi-mutation diff. I didn't want it to create separate "Added token" lists in that case. Do you think that situation is possible?

Reply:
Ah ok, from my exploring of data that's definitely possible :)